### PR TITLE
COASTAL-618 Support events originating from Pump UI

### PR DESCRIPTION
**Pull request by ps2**
https://tidepool.atlassian.net/browse/COASTAL-618

### CODE REVIEW

File: LoopKit/CarbKit/CarbStore.swift [hunk 1]
```diff
@@ -351,7 +351,7 @@ public final class CarbStore: HealthKitSampleStore {
                 return
             }
 
-            self.handleUpdatedCarbData(updateSource: .queriedByHealthKit)
```
Comment (rickpasetto) [inline LoopKit/CarbKit/CarbStore.swift:354]: [nit] Just curious...why was this removed?  Not a blocker just wondering...

**Review by rickpasetto — APPROVED** [review]
LGTM. Looks like you deleted the old models & mapping files.  I'm glad, but I just want to make sure it was intentional...

File: LoopKit/CarbKit/CarbStore.swift (hunk 1 above)
Comment (ps2) [reply LoopKit/CarbKit/CarbStore.swift:354]: The update source wasn't being used anywhere, so just cleanup. I think it used to be used by uploading code, but that has changed so much.

**Comment by ps2** [comment]
Yes, removing the old models was intentional.  Version 1 is needed to support the current release of DIY loop. Versions 2 and 3 never made it into any release, and so were just cruft.
